app/views/login/canvas/mobile_login.html.erb

```diff
@@ -75,6 +75,15 @@
       <a class="forgot-password flip-to-back"><%= t('dont_know_password', "Forgot Password?") %></a>
     <% end %>
   </div>
+  <footer role="contentinfo" id="footer" class="ic-Login-footer">
+    <div id="footer-links" class="ic-Login-footer__links">
+      <a class="open-source-link" href="https://github.com/strongmind/canvas-lms">
+        <%= t('open_source_learning_management_system', 'Modified Open Source LMS') %>
```

Comment:
Given the screen limitation for mobile I am wondering about what folks think about leaving this off.

Reply:
I am not worried about it from a design perspective. It takes up very minimal space and does not come close to other content on the 8 device sizes I've tested. If the business deems it a priority to remove then I can do that easily, otherwise this has been specifically asked to retain on the mobile view by Bovi.

Reply:
this should be on both login views.

Reply:
ok, I will go ahead and merge this PR then so we can push it next build.